tools/importer-msgraph-metadata/components/parser/resourceids.go

```diff
@@ -360,6 +315,9 @@ const (
 	SegmentFunction       ResourceIdSegmentType = "Function"
```

Comment:
Unrelated to this PR, but I noticed all of these new segment types which I don't believe we have in the Resource Manager Importer? I'd find it helpful if we could briefly go over these in the upcoming handovers.

Reply:
Sure thing! These are mapped to 'static' segments in the API definitions and resulting SDK, but we need to distinguish between them during parsing.